corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -199,6 +199,7 @@ hqDefine('app_manager/js/releases/releases', function () {
             hqImport('analytix/js/kissmetrix').track.event('Clicked Deploy');
             $.post(releasesMain.reverse('hubspot_click_deploy'));
             self.get_short_odk_url();
+            self.releaseLocked(true);
```

Comment:
why do we need to lock things when someone clicks on "Publish"?

Reply:
Actually this is erroring since `releaseLocked` is not present on this object.

Reply:
Cleaned up `releaseLocked`